### PR TITLE
Use SentryInterceptor in gRPC servers

### PR DESCRIPTION
**Pull request by jotare**
### Description
- Add `SentryInterceptor` to gRPC servers to automatically send unhandled sentry issues

### How was this PR tested?
Describe how you tested this PR.

### CODE REVIEW

**Comment by jotare** [comment]
Also [sc-6814]

File: nucliadb_utils/nucliadb_utils/grpc.py [hunk 1]
```diff
@@ -23,7 +23,9 @@
 
 from grpc import aio  # type: ignore
 from grpc import ChannelCredentials
+
 from nucliadb_telemetry.grpc import GRPCTelemetry
+from nucliadb_telemetry.grpc_sentry import SentryInterceptor
```
Comment (vangheem) [inline nucliadb_utils/nucliadb_utils/grpc.py:28]: we'll need a new release of the nucliadb-telemetry package first in practice so we will need to split this into a separate PR.

File: nucliadb_utils/nucliadb_utils/grpc.py [hunk 2]
```diff
@@ -78,7 +80,10 @@ def get_traced_grpc_server(service_name: str, max_receive_message: int = 100):
     tracer_provider = get_telemetry(service_name)
     if tracer_provider is not None:  # pragma: no cover
         otgrpc = GRPCTelemetry(f"{service_name}_grpc", tracer_provider)
-        server = otgrpc.init_server(max_receive_message=max_receive_message)
+        server = otgrpc.init_server(
+            max_receive_message=max_receive_message,
+            interceptors=[SentryInterceptor()],
```
Comment (vangheem) [inline nucliadb_utils/nucliadb_utils/grpc.py:85]: This should only be conditional to sentry being configured right?

**Review by vangheem — CHANGES_REQUESTED** [review]

File: nucliadb_utils/nucliadb_utils/grpc.py (hunk 2 above)
Comment (jotare) [reply nucliadb_utils/nucliadb_utils/grpc.py:85]: I'll change it to use `nucliadb_telemetry.errors.capture_exception`, that already handles this

**Review by vangheem — APPROVED** [review]
